lua/entities/gmod_wire_expression2/core/vector2.lua

```diff
@@ -1096,8 +1096,8 @@ __e2setcost(7)
 -- Returns a random vector4 between -1 and 1
```

Comment:
That comment doesn't describe what's happening. It should at least describe the properties of the returned vectors and their distribution properties